_pages/security/scanning.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Security Scanning
+navtitle: Scanning
+parent: Security
+---
+
+Security scanning is broken down in several areas:
+
+* [Static](../static-analysis/)
+* [Dynamic](../dynamic-scanning/)
+    * Active
+        * Infrastructure-level scan (done at the cloud.gov level)
+        * Penetration test (required if dealing with sensitive information)
+    * Passive
+
+## Alerts
+
+Alerts come in one of four categorizations:
+
+* **Low** and **Medium** can be resolved after the ATO is granted.
+* **High** and **Critical** must be resolved before the ATO is granted.
+
+If any findings are from human testing or are critical vulnerabilities, you must fix or explain as a false positive.
```

Comment:
That sounds good. All findings that you determine are false positives should be documented as such.